react/features/aeternity/utils.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Check if the string is an aeternity account nubmer or chain name.
+ *
+* @param {string} str - String to check.
+* @returns {boolean}
+ */
+export function isAccountOrChainName(str) {
+    return str.startsWith('ak_') || str.endsWith('.chain');
```

Comment:
can we use a regex, as in adress and .chain names there are some requirements (alphanumeric and length). For instance if i use cyrillic and put .chain it will not be a valid one also ak_abc is too short as well